Gems/EditorPythonBindings/Code/Source/PythonUtility.cpp

```diff
@@ -845,6 +855,8 @@ namespace EditorPythonBindings
                     type = "bool";
                 }
                 else if (
+                    AZ::AzTypeInfo<short>::Uuid() == typeId || AZ::AzTypeInfo<unsigned short>::Uuid() == typeId ||
```

Comment:
repeat: The check for `short` and `unsigned short` should be removed as they are already being checked via `AZ::s16` and `AZ::u16`.